test/loadtime/payload/payload.proto

```diff
@@ -14,5 +14,6 @@ message Payload {
   uint64                    size        = 3;
   google.protobuf.Timestamp time        = 4;
   bytes                     id          = 5;
-  bytes                     padding     = 6;
+  uint32                    lane        = 6;
```

Comment:
This is a breaking change that only affects e2e tests and QA. I added `lane` in position 6, and moving `padding` to 7, because I think `padding` needs to go last.

Reply:
I think we don't need padding to go last, it'll work either way

Reply:
Ok, then I'll leave them in the original order.